packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -48,6 +48,19 @@
         "language": "apex"
       }
     ],
+    "configuration": {
+      "type": "object",
+      "title": "%configuration_title%",
+      "properties": {
+        "salesforcedx-vscode-apex-debugger.connectionTimeoutMs": {
+          "type": "number",
+          "default": 20000,
+          "minimum": 500,
+          "maximum": 120000,
+          "description": "%connection_timeout_ms_description%"
```

Comment:
IDE1:

![ide1debuggerconnectiontimeout](https://user-images.githubusercontent.com/7286437/31145385-79fd6e14-a838-11e7-8921-7522967af5ea.png)

VSC:

![debuggerconnectiontimeout](https://user-images.githubusercontent.com/7286437/31145541-de60df94-a838-11e7-8031-3fb259a4b2a9.png)
![debuggerconnectiontimeoutdefault](https://user-images.githubusercontent.com/7286437/31145540-de604ab6-a838-11e7-8b49-a2df3a8a0bde.png)
![debuggerconnectiontimeoutabovemax](https://user-images.githubusercontent.com/7286437/31145539-de5f00a2-a838-11e7-8b11-36d906553df7.png)

Reply:
@ruthemmanuelle , let me know if you'd like to see something different in the hover text in the first VSC screenshot.

Reply:
Maybe put "in milliseconds" in parentheses? It would make the text easier to read.